### ISSUE TITLE
Text contrast is not always accessible

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/contacts#5049]
### Describe the bug

<img width="482" height="218" alt="Image" src="https://github.com/user-attachments/assets/f52c9ad9-c32e-4057-b838-3c2e8b8b3294" />

### Steps to reproduce

1. Go to contact without attachments

### Expected behavior

Darker text

### Actual behavior

Lighter text, not accessible

It's possible other parts of the app are also affected by this issue, this is for all of the cases

### PHP engine version

None

### Web server

None

### Database

None
